OpenRA.Game/Traits/Player/Shroud.cs

```diff
@@ -315,6 +321,7 @@ public void ExploreAll()
 				{
 					touched[puv] = true;
 					explored[puv] = true;
+					RevealedCells++;
```

Comment:
I expect this and the one below will also break the count: we are already updating in `Tick` so shouldn't need to do anything in these.